inference-engine/thirdparty/clDNN/src/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -396,6 +396,8 @@ void prepare_primitive_fusing::fuse_simple_primitives(program_impl &p) {
             p.fuse_nodes(input_data, scale_node);
         };
 
+        auto eltwise_supports_fusings = [](eltwise_node& node) -> bool { return true; };
```

Comment:
Formal parameter 'node' is unreferenced in lambda's body. This warning is treated as error and it breaks Windows build in CI.  We should either put some logic inside that actually needs 'node', or completely remove this lambda.

Reply:
just removed this check.